examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -235,6 +235,14 @@
     "--checkpoints_to_keep", default=1, type=int, help="The number of last checkpoints to keep",
 )
 
+parser.add_argument(
+    "--add_carry_value", action="store_true", help="Add a carry-over value to all categorical slots.",
+)
+
+parser.add_argument(
+    "--add_carry_status", action="store_true", help="Add a carry-over status to all categorical slots.",
+)
```

Comment:
Maybe make both --add_carry_value and --add_carry_status True by default?

Reply:
done.